### DISCUSSION TITLE
Installazione Gratuita

### DISCUSSION THREAD

**Discussion by NikitaTrofymchuk** [iisgiua/giuaschool#504] [Idee e proposte]
Buonasera , sono Nikita Trofymchuk 
è vi comunico che ad ogni richiesta per Installare il Registro Giua@School potrete contattare la seguente mail:
n.trentini.icempoliest@gmail.com
vi risponderò è vi installerò il Registro "GRATUITAMENTE"

Cordiali Saluti , 
Nikita Trofymchuk
@trinko Cosa ne pensa?

**Comment by trinko** [comment]
E' senz'altro un'ottima iniziativa. Grazie.

**Reply by NikitaTrofymchuk** [reply]
di nulla

**Comment by giadaschiavone** [comment]
Ciao Nicola, bell'idea.
Scrivimi quando avrai bisogno! 😉

**Reply by NikitaTrofymchuk** [reply]
ok

**Comment by fabriziotoppi** [comment]
Ciao Nicola  a me servirebbe aiuto per l'installazione. Ti ho già scritto tramite email

**Reply by luigirotondi2009** [reply]
Ha bisogno di una mano? Io ci sono.

**Comment by dirigenteic-demo** [comment]
Vorrei avere un tutorial per istallare giuaschool.

**Reply by luigirotondi2009** [reply]
Ha bisogno di una mano? Io ci sono.
